cwl_tes/main.py

```diff
@@ -123,13 +132,18 @@ def main(args=None):
 
     if parsed_args.token:
         try:
+            header = jwt.get_unverified_header(parsed_args.token)
+            token_public_key = load_public_key(header.get('kid'))
```

Comment:
If the header has no `kid` value, I think this gets called as `load_public_key(None)` and would raise `TypeError` inside the function.

Reply:
I now check for a 'kid' first. If it is missing, it raises an error, because then it is not a valid token anyway.